common/test/acceptance/pages/studio/textbook_upload.py

```diff
@@ -59,6 +60,13 @@ def click_textbook_submit_button(self):
         self.q(css='#edit_textbook_form button[type="submit"]').first.click()
         self.wait_for_element_absence(".wrapper-form", "Add/Edit form closed")
 
+    def click_view_live_link(self):
+        """
+        Click view live link
+        """
+        element = self.q(css='a.view')[0]
+        ActionChains(self.browser).move_to_element(element).click(element).perform()
```

Comment:
There is no need to explicitly reference ActionChains. Just do something like this--

self.q(css='#edit_textbook_form button[type="submit"]').first.click()